superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -194,7 +201,7 @@ const DashboardBuilder: FC<DashboardBuilderProps> = () => {
                     shouldFocus={shouldFocusTabs}
                     menuItems={[
                       <IconButton
-                        className="fa fa-level-down"
+                        icon="FallOutlined"
```

Comment:
Just an idea. Wouldn't be better to pass the icon down directly instead of the name reference? In that way, if I want a button to have an icon of a smaller size or a different color, I can do it straight.

Reply:
That's a nice idea! I'll make the change.